proto/lskvserver.proto

```diff
@@ -66,3 +66,31 @@ message GetReceiptResponse
   etcdserverpb.ResponseHeader header = 1;
   Receipt receipt = 2;
 }
+
+message TxStatusRequest
+{
+  int64 revision = 1;
+  uint64 raft_term = 2;
+}
+
+message TxStatusResponse
+{
+  etcdserverpb.ResponseHeader header = 1;
+  enum Status {
```

Comment:
Link to https://microsoft.github.io/CCF/main/use_apps/verify_tx.html#checking-for-commit perhaps?